### PR TITLE
Announce release of r2dii.analysis 0.1.2

### PR DESCRIPTION
**Pull request by jdhoffa**
@maurolepore this is just a draft of the post explaining the r2dii.analysis 0.1.2 release. Would like you to give it a look over when you can, but would also like to have George read it over before publishing!

### CODE REVIEW

File: _posts/2020-12-07-r2dii-analysis-0-1-2-is-now-on-cran/r2dii-analysis-0-1-2-is-now-on-cran.Rmd [hunk 1]
```diff
@@ -0,0 +1,172 @@
+---
```
Comment (maurolepore) [inline _posts/2020-12-07-r2dii-analysis-0-1-2-is-now-on-cran/r2dii-analysis-0-1-2-is-now-on-cran.Rmd]: It'll be nice to have a `preview:` field pointing to an (CCO) image -- maybe something about bugs and holidays?

File: _posts/2020-12-07-r2dii-analysis-0-1-2-is-now-on-cran/r2dii-analysis-0-1-2-is-now-on-cran.Rmd [hunk 2]
```diff
@@ -0,0 +1,172 @@
...
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(
+  echo = TRUE,
+  collapse = TRUE,
+  comment = "#>"
+)
+```
+
+r2dii.analysis 0.1.2 is now on CRAN. This release fixes a number of bugs. All changes are listed [here](https://2degreesinvesting.github.io/r2dii.analysis/news/index.html); this post highlights the major bugs fixes.
```
Comment (maurolepore) [inline _posts/2020-12-07-r2dii-analysis-0-1-2-is-now-on-cran/r2dii-analysis-0-1-2-is-now-on-cran.Rmd]: I reworded this quite a bit.

File: _posts/2020-12-07-r2dii-analysis-0-1-2-is-now-on-cran/r2dii-analysis-0-1-2-is-now-on-cran.Rmd [hunk 3]
```diff
@@ -0,0 +1,172 @@
...
+
+To use r2dii.analysis, you can load it into your active R session with `library()`. This example uses some other packages that you may also load now so they are easy to see.
+
+```{r}
+library(dplyr)
+library(ggplot2)
+library(r2dii.data)
+library(r2dii.match)
+library(r2dii.analysis)
+```
```
Comment (maurolepore) [inline _posts/2020-12-07-r2dii-analysis-0-1-2-is-now-on-cran/r2dii-analysis-0-1-2-is-now-on-cran.Rmd]: I grouped all calls to `library()`.

File: _posts/2020-12-07-r2dii-analysis-0-1-2-is-now-on-cran/r2dii-analysis-0-1-2-is-now-on-cran.Rmd [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+)
+```
+
+### r2dii.analysis 0.1.2
+
+There are a few major fixes in this release of r2dii.analysis, in particular to the functions `target_market_share` and `target_sda`. 
+
+#### target_market_share()
+
+`target_market_share` outputs a data frame with the value `technology_share`. This value is expected to always sum to `1`, even when aggregated across appropriate groupings (e.g. by `sector`, `region`, `metric`). Before, this was true only for results at the level of companies but not portfolios. Now, this is fixed and `technology_share` sums `1` at both levels (at least to 10 significant digits; further digits may be off due to rounding errors):
```
Comment (maurolepore) [inline _posts/2020-12-07-r2dii-analysis-0-1-2-is-now-on-cran/r2dii-analysis-0-1-2-is-now-on-cran.Rmd]: I reworded this quite a bit.

**Review by maurolepore — APPROVED** [review]
Thanks!

**Comment by jdhoffa** [comment]
I know you already asked me, and I think I said no, but do you think it's worth mentioning the updates in `r2dii.data` and `r2dii.match`?
FYI: 

r2dii.data 0.1.6

- region_isos now correctly maps Kosovo iso to “xk” (@cjyetman #186).
- region_isos now includes more missing isos for all global regions (#183).
- Add missing sectors to nace_classification (@georgeharris2deg #171).
- Fix borderline definitions in isic_classification and nace_classification (@georgeharris2deg #163).
- New dataset cnb_classification (@georgeharris2deg #173).

r2dii.match 0.0.7

- match_name() gains ... to pass additional arguments to stringdist::stringsim() (@evgeniadimi #310).

**Comment by jdhoffa** [comment]
and if so, should I just add to this post?

**Comment by maurolepore** [comment]
Yeah, I think all changes deserve to be announced -- but changes in, e.g. r2dii.match name are too few to deserve its own post. If you see a way to announce all here, then I support that.

**Comment by maurolepore** [comment]
I did minor edits and change the release number to reflect the upcomming patch. 

I fail to build the site locally. Not sure what's going on.

**Comment by jdhoffa** [comment]
Hmm shit ok. Well we'll need to wait until 0.1.3 is accepted to CRAN anyhow.

**Comment by maurolepore** [comment]
My problem seems unrelated to your post. Removing the folder still causes this error

```r
Error in file(con, "r") : cannot open the connection
Calls: <Anonymous> ... tagList -> dots_list -> includeCSS -> readLines -> file
In addition: Warning message:
In file(con, "r") :
  cannot open file '/home/mauro/git/2degreesinvesting.github.io/flatly': No such file or directory
Execution halted

Exited with status 1.
```

Maybe I need to install something that comes with the flatly theme. Are you able to build the site?

r2dii.analysis should be on CRAN in a few minutes. It has no reverse dependencies so it should fly quickly.

**Comment by jdhoffa** [comment]
Ya i had actually received that issue as well. The thing is, I'm pretty sure the style of the site is determined by `style.css` and not by `flatly` at all. In fact, I think if you remove that line, it builds the site in the usual style. 

I would suggest we do that for now, perhaps in a separate PR, and see how it looketh

**Comment by jdhoffa** [comment]
(I was actually fucking around on a totally separate branch with the new styling capabilities offered in `distill 1.0.0` which is how i found that. Maybe the error comes from updating to `1.0.0`)

**Pull request merged**